File: python_t3_full/Clone_2084.py
def getPrimes(n) :
	yield 2
	i = 3
	while i < n :
		for a in getPrimes(int(math.sqrt(i)) + 1) :
			if i % a == 0 :
				break
		else :
			yield i
		i += 2


def getPrimes(n):
    yield 2
    i = 3
    while i < n:
        for a in getPrimes(int(i ** 0.5) + 1):
            if not i % a == 0:
                yield i
        i += 2

# 8

import math

File: java_t3_full/Clone_2084.java
public static double sqrt (int n) {
    double low = 0;
    double high = n;
    double mid = (high - low) / 2;
    while (Math.abs ((mid * mid) - n) > 0.000000000001) {
        if ((mid * mid) > n) {
            high = mid;
            mid = (high - low) / 2;
        } else {
            low = mid;
            mid = mid + ((high - low) / 2);
        }
    }
    return mid;
}


 public static double sqrt(int n) 
{ 
    double left = 0; 
    double right = n; 
  
    while(Math.abs(right - left) > 0.00000000001) 
    { 
        double mid = left + (right - left)/2; 
        double sq = mid * mid; 
  
        if(sq > n) { 
            right = mid; 
        } 
        else { 
            left = mid; 
        } 
    } 
    return left; 
} 


